components/endpoint-configuration-management/org.wso2.carbon.identity.api.server.endpoint.configuration.management/org.wso2.carbon.identity.api.server.endpoint.configuration.management.v1/src/main/resources/choreo.yaml

```diff
@@ -0,0 +1,400 @@
+openapi: 3.0.0
+info:
+  title: WSO2 Identity Server - Endpoint Configuration Manager API Definition
+  description: This document specifies a **RESTful API** for **WSO2 Identity Server endpoints configurations**
+  contact:
+    name: WSO2
+    url: http://wso2.com/products/identity-server/
+    email: architecture@wso2.org
```

Comment:
since this mailing list is deprecated, shall we use iam-dev@wso2.org instead?

Reply:
resolved